lib/sdnc

```diff
@@ -6,7 +6,7 @@ source /var/onap/asserts
 
 # build_sdnc_images() - Builds SDNC images from source code
 function build_sdnc_images {
-    local src_folder=/opt/sdnc/oam
+    local src_folder=/opt/openecomp/sdnc/oam
```

Comment:
Lines 9-11 should be replaced for 

local src_folder=$1

The reason behind that is because are repeated on 30, 31 and 33, so when the build_sdnc_images function is called on line 35, we can pass $src_folder as an argument.